Parsedown.php

```diff
@@ -531,16 +502,30 @@ protected function blockHeader($Line)
 
     protected function blockList($Line)
     {
-        list($name, $pattern) = $Line['text'][0] <= '-' ? array('ul', '[*+-]') : array('ol', '[0-9]+[.\)]');
+        list($name, $pattern) = $Line['text'][0] <= '-' ? array('ul', '[*+-]') : array('ol', '[0-9]{1,9}[.\)]');
 
-        if (preg_match('/^('.$pattern.'[ ]+)(.*)/', $Line['text'], $matches))
+        if (preg_match('/^('.$pattern.'([ ]+|$))(.*)/', $Line['text'], $matches))
```

Comment:
I'd guess that allowing the string to end, instead of a space being present in the regex here would be the cause of the failure mentioned above? (in https://github.com/aidantwoods/parsedown/pull/2#issuecomment-253655219)